.github/workflows/feedstock-pr.yml

```diff
@@ -0,0 +1,29 @@
+name: Open-CE Feedstock PR Tests
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  optional_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: '0'
+      - uses: conda-incubator/setup-miniconda@v2.0.0
+        with:
+          auto-update-conda: true
+          python-version: "3.7"
```

Comment:
Should we switch to Python 3.9 for these tests? Given that 3.7 is somewhat "old"...

Reply:
Hi @alexlang74  , Post V1.5.1 we shall add this update for python-version 3.9